synapse/python_dependencies.py

```diff
@@ -107,7 +107,7 @@
     "systemd": ["systemd-python>=231"],
     "url_preview": ["lxml>=3.5.0"],
     "sentry": ["sentry-sdk>=0.7.2"],
-    "opentracing": ["jaeger-client>=4.0.0", "opentracing>=2.2.0"],
+    "opentracing": ["jaeger-client>=4.0.0,<4.7.0", "opentracing>=2.2.0"],
```

Comment:
Looks like this change was accidentally included?

Reply:
Oh, I needed to back this out. Good call.